blocks/api/serializer.js

```diff
@@ -50,20 +50,28 @@ export function getSaveContent( blockType, attributes ) {
 	}
 
 	// Adding a generic classname
-	const addClassnameToElement = ( element ) => {
-		if ( ! element || ! isObject( element ) || ! className ) {
+	const addGenericAttributes = ( element ) => {
```

Comment:
I like this, I think it looks good. I don't think it's a _high priority_, it seems like an optimization. But I'm a fan of the idea and the look.